README.md

```diff
@@ -105,7 +105,7 @@ Note that by setting `root` to true, we ensure that no ancestor configuration is
 
 ## Tests
 
-`$ npm test`   
```

Comment:
Can you restore back the empty spaces so that they are in two separate lines?

Reply:
ops. will do.

Reply:
will rebase this as well

Reply:
done